src/pool/srv_pool.c

```diff
@@ -1064,7 +1064,7 @@ read_db_for_stepping_up(struct pool_svc *svc, struct pool_buf **map_buf,
 			 */
 			D_DEBUG(DB_MD, DF_UUID": new db\n",
 				DP_UUID(svc->ps_uuid));
-			rc = +DER_UNINIT;
+			rc = + DER_UNINIT;
```

Comment:
The "+" was deliberately added to prevent readers from thinking a "-" is forgotten. Adding a space is incorrect.

The canonical doc for this error is

	/**
	 * Step up to be the new leader. If the DB is new (i.e., has not been
	 * bootstrapped), return +DER_UNINIT.
	 */
	int (*sc_step_up)(struct ds_rsvc *svc);

If necessary we could add duplicated comments to read_db_for_stepping_up.

Reply:
Again, this came from a commit hook and wasn't intended to be included.

I don't see why a negative value can't be used here though, and there looks to be one more case of DER_UNITIT in this file and one in src/rsvc/srv.c where DER_UNITIT is used without a + or a -.